packages/cli/src/scripts/add.ts

```diff
@@ -0,0 +1,15 @@
+import ControllerFor from '../models/local/ControllerFor';
+import ZosPackageFile from '../models/files/ZosPackageFile';
+import { AddParams } from './interfaces';
+
+export default function add({ contractsData, packageFile }: AddParams): void | never {
+  if (contractsData.length === 0) throw new Error('At least one contract name must be provided to add.');
+  contractsData = contractsData.map((data) => typeof(data) === 'string' ? { name: data, alias: data } : data);
+
+  const controller = ControllerFor(packageFile);
```

Comment:
`controller` would be a LocalController here right? It may be nice to add the explcity type for easier reading of the code.

Reply:
We thought the same thing! I was planning to type all `ControllerFor`s, but as we talked offline, I'll remove both local and network `ControllerFor` ASAP.